mobile/native/src/trade/order/handler.rs

```diff
@@ -17,9 +17,12 @@ pub async fn submit_order(order: Order) -> Result<()> {
     let url = format!("http://{}", config::get_http_endpoint());
     let orderbook_client = OrderbookClient::new(Url::parse(&url)?);
 
-    if let Err(e) = position::handler::update_position_after_order_submitted(order) {
-        order_failed(Some(order.id), FailureReason::OrderNotAcceptable, e)?;
-        bail!("Could not submit order because extending/reducing the position is not part of the MVP scope");
+    let position = position::handler::get_current_position()?;
+
+    if !position::handler::is_order_acceptable_for_position(&position, &order)? {
```

Comment:
🤔 We are handling that scenario also on the coordinator. Handling this in the app definitely makes sense, but will also mean that we will not see this trade attempt in our unsuccessful trades statistics. So maybe its better to not check on the app side? as this should anyways not happen?